cypress/integration/checkout/checkout.multipiece.items.spec.js

```diff
@@ -0,0 +1,152 @@
+import TopMenu from '../../support/fragments/topMenu';
+import getRandomPostfix from '../../support/utils/stringTools';
+import Helper from '../../support/fragments/finance/financeHelper';
+import CheckOutActions from '../../support/fragments/check-out-actions/check-out-actions';
+import NewServicePoint from '../../support/fragments/service_point/newServicePoint';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints';
+import ConfirmMultipieceCheckOutModal from '../../support/fragments/checkout/confirmMultipieceCheckOutModal';
+import SwitchServicePoint from '../../support/fragments/service_point/switchServicePoint';
+
+describe('Check Out', () => {
+  let user = {};
+  const FIRST_ITEM_BARCODE = Helper.getRandomBarcode();
+  const SECOND_ITEM_BARCODE = Helper.getRandomBarcode();
+  const THIRD_ITEM_BARCODE = Helper.getRandomBarcode();
+  const FOURTH_ITEM_BARCODE = Helper.getRandomBarcode();
+  let userBarcode = '';
+  const instanceTitle = `autotest_instance_title_${getRandomPostfix()}`;
+  const quantityPiecesForSecondItem = '3';
+  const quantityPiecesForThirdItem = '2';
+  const quantityPiecesForFourthItem = '1';
+  const quantityOfMissingPieces = '2';
+  const descriptionOfPiece = `autotest_description_${getRandomPostfix()}`;
+  const missingPieceDescription = `autotest_description_${getRandomPostfix()}`;
+  let servicePoint;
+
+  beforeEach(() => {
+    cy.createTempUser([
+      permissions.checkoutCirculatingItems.gui,
+      permissions.uiUsersView.gui,
+      permissions.uiUserEdit.gui
+    ])
+      .then(userProperties => {
+        user = userProperties;
+        servicePoint = NewServicePoint.getDefaulServicePoint();
+        ServicePoints.createViaApi(servicePoint.body);
+        cy.addServicePointToUser([servicePoint.body.id],
+          user.userId, servicePoint.body.id);
+      })
+      .then(() => {
+        cy.login(user.username, user.password);
+        cy.visit(TopMenu.inventoryPath);
+        cy.getAdminToken()
+          .then(() => {
+            cy.getLoanTypes({ limit: 1 });
+            cy.getMaterialTypes({ limit: 1 });
+            cy.getLocations({ limit: 2 });
+            cy.getHoldingTypes({ limit: 2 });
+            cy.getHoldingSources({ limit: 2 });
+            cy.getInstanceTypes({ limit: 1 });
+            cy.getUsers({ limit: 1, query: `"personal.lastName"="${user.username}" and "active"="true"` })
+              .then((users) => {
+                userBarcode = users[0].barcode;
+              });
+          })
+          .then(() => {
+            cy.createInstance({
+              instance: {
+                instanceTypeId: Cypress.env('instanceTypes')[0].id,
+                title: instanceTitle,
+              },
+              holdings: [{
+                holdingsTypeId: Cypress.env('holdingsTypes')[0].id,
+                permanentLocationId: Cypress.env('locations')[0].id,
+                sourceId: Cypress.env('holdingSources')[0].id,
```

Comment:
try to redesign existing code from cypress custom command to fragment method

Reply:
Done